commands/commands.go

```diff
@@ -99,6 +101,19 @@ func (c *Command) Run(pctx context.Context, bus *events.EventBus) {
 			bus.Publish(events.Event{events.Error, err.Error()})
 			return
 		}
+
+		// if we're able to, log the PID of our Command's exec process through
+		// our logger fields
+		if c.Cmd != nil && c.Cmd.Process != nil {
+			pid := c.Cmd.Process.Pid
+			if len(c.fields) > 0 {
+				c.fields["pid"] = pid
+			} else {
+				c.fields = log.Fields{"pid": pid}
+			}
+			c.logger = *log.WithFields(c.fields)
```

Comment:
I'm pretty sure this breaks the "raw" logging, doesn't it? I think we only want to assign to `c.logger` in the case where `c.fields > 0`, so this line should be in first branch of the `if` above.